tests/json.py

```diff
@@ -182,6 +182,21 @@ def foo():
     assert response_dict["count"] == 3
 
 
+def test_stupeflask_response_with_paginate_headers(json_app, client):
+    encoder_rules.append((Cursor, lambda c: c.data))
+
+    @json_app.route("/")
+    @paginate(limit=1, skip=0, headers=True)
+    def foo():
+        return Cursor([1, 2, 3])
+
+    response = client.get("/")
+    assert response.status_code == 200
+
+    assert "X-Total-Count" in response.headers
```

Comment:
```suggestion
    assert response.headers["X-Total-Count"] == 3
```

Reply:
Do we really need both tests then?

Reply:
In my opinion yes, this one validates the code that actually adds the headers to the `response` object (and without it coverage is not 100% :clown_face: )

Reply:
Right. But then a similar test should be done in `tests/app.py` then, because we want the headers mode to work also with the non-json Stupeflask version.